developer-docs/bolt-api-servers.md

```diff
@@ -165,9 +166,18 @@ The metadata object is optional, and contains metadata about the task being run.
 - `parameters`: Object, *optional* - A JSON object whose keys are parameter names, and whose values are JSON objects with 2 keys:
     - `description`: String, *optional* - The parameter description.
     - `type`: String, *optional* - The type the parameter should accept.
+    - `sensitive`: Boolean, *optional* - Whether the task runner should treat the parameter value as sensitive
+    - `input_method`: String, *optional* - What input method should be used to pass params to task (stdin, environment, powershell)
 
 #### Files
-# TODO
+The files array is required, and contains details about the files the task needs as well as how to get them. Array items should be objects with the following keys:
+- `uri`: Object, *required* - Information on how to request task files
+    - `path`: String, *required* - Relative URI for requesting task content
+    - `params`: Object, *required* - Query parameters for locating task data
+        - `environment`: String, *required* - Environment task files are in
+- `sha256`: String, *required* - Shasum of the file contents
+- `filename`: String, *required* - File name including extension
+- `size`: Number, *optional* - Size of file in Bytes
```

Comment:
I forget - do we actually do anything with this parameter?

Reply:
Eh it isn't hurting anything - the bigger question is do we want to add a size check somewhere if this is present? I don't see where in bolt server or puppetserver that check happens, but maybe I'm missing it.